packages/wdio-types/src/Capabilities.ts

```diff
@@ -161,6 +161,8 @@ export interface VendorExtensions extends EdgeCapabilities, AppiumW3CCapabilitie
      * @private
      */
     'browserstack.wdioService'?: string
+    'browserstack.buildIdentifier'?: string
+    'browserstack.localIdentifier'?: string
```

Comment:
These are not valid capabilities , I would suggest to not add them.

Reply:
@christian-bromann adding this for backward compatibility as v7 version of browserstack-service is compatible with webdriverIO v5. I'll remove these types from main branch PR.

So, user can pass 

```
capabilities: [{
    browserName: 'chrome',
   'browserstack.buildIdentifier': '123'
    // ...
}]
```